packages/emitter-framework/src/python/components/class-declaration/class-declaration.test.tsx

```diff
@@ -0,0 +1,855 @@
+import { getOutput } from "#python/test-utils.jsx";
+import { Tester } from "#test/test-host.js";
+import { List } from "@alloy-js/core";
+import * as py from "@alloy-js/python";
+import { t } from "@typespec/compiler/testing";
+import { describe, expect, it } from "vitest";
+import { ClassDeclaration } from "../../../../src/python/components/class-declaration/class-declaration.js";
+import { ClassMethod } from "../../../../src/python/components/class-declaration/class-method.js";
+import { EnumDeclaration } from "../../../../src/python/components/enum-declaration/enum-declaration.js";
+
+describe("Python Class from model", () => {
+  it("creates a class", async () => {
+    const { program, Widget } = await Tester.compile(t.code`
+
+    model ${t.model("Widget")} {
+      id: string;
+      weight: int32;
+      aliases: string[];
+      isActive: boolean;
+      color: "blue" | "red";
+      promotionalPrice: float64;
+      description?: string = "This is a widget";
+      createdAt: int64 = 1717334400;
+      tags: string[] = #["tag1", "tag2"];
+      isDeleted: boolean = false;
+      alternativeColor: "green" | "yellow" = "green";
+      price: float64 = 100.0;
+    }
+    `);
+
+    expect(getOutput(program, [<ClassDeclaration type={Widget} />])).toRenderTo(
+      `
+          from dataclasses import dataclass
+          from typing import Literal
+
+          @dataclass
+          class Widget:
+            id: str
+            weight: int
+            aliases: list[str]
+            is_active: bool
+            color: Literal["blue", "red"]
+            promotional_price: float
+            description: str = "This is a widget"
+            created_at: int = 1717334400
+            tags: list[str] = ["tag1", "tag2"]
+            is_deleted: bool = False
+            alternative_color: Literal["green", "yellow"] = "green"
+            price: float = 100.0
+          
+          `,
+    );
+  });
+
+  it("declares a class with multi line docs", async () => {
+    const { program, Foo } = await Tester.compile(t.code`
+      /**
+       * This is a test
+       * with multiple lines
+       */
+      model ${t.model("Foo")} {
+        knownProp: string;
+      }
+    `);
+
+    expect(getOutput(program, [<ClassDeclaration type={Foo} />])).toRenderTo(
+      `
+          from dataclasses import dataclass
+
+          @dataclass
+          class Foo:
+            """
+            This is a test
+            with multiple lines
+            """
+            known_prop: str
```

Comment:
There should be a blank line here to conform to [PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings):
> Insert a blank line after all docstrings (one-line or multi-line) that document a class – generally speaking, the class’s methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.